### DISCUSSION TITLE
Access point not working

### DISCUSSION THREAD

**Discussion by roadsterm** [rern/rAudio#1570] [Fixed Issues]
I've tried to activate Access Point (AP ) in order to play rAudio in my car (no wire or wifi network) but I've no luck on both 20231128 and 20240113 versions running in Pi 4b rev 1.5.

1) When connect to router via wifi I can't turn on the AP, the message " wifi is connecting" is shown

2) when disconnect  wifi or turn off wifi and use wire connection, I can turn on AP,   and key in IP address and password in pop up box but after l click OK button on pop up box for a few second, the AP turn off.

3) When wire and wireless are not available, the AP is not automatic on.

 Not sure, it is a bug or incorrect setting.

Any advice is very appreciated 

Thanks

**Comment by cmh714** [comment]
I think the problem is #2. If I read correctly, number 1 wont work because the wifi can only be used for one thing at a time.

Since AP wont stay ON thats why you cant connect on number 3. Number 2 needs o be addressed imho.....seems like a bug since I cannot get AP to stay ON either.

**Reply by roadsterm** [reply]
@cmh714 
Thank you so much for your explanation.

**Comment by roadsterm** [comment]
@rern 
Could you please fix this bug.
Thanks

**Reply by rern** [reply]
Next update will switch from `hostapd` + `dnsmasq` to `iwd` which is better for the Access Point feature.

**Reply by roadsterm** [reply]
Thank you very much for your kindness.

**Reply by roadsterm** [reply]
It would be great, if AP is turned on at the first boot so wire connection is not necessary at all.

Thanks

**Reply by roadsterm** [reply]
After update to version 20240122, the AP works well.

Thanks for your great work.
